base/nullable.jl

```diff
@@ -59,7 +59,20 @@ otherwise, returns `y` if provided, or throws a `NullException` if not.
     end
 end
 
-get(x::Nullable) = isnull(x) ? throw(NullException()) : x.value
+"""
+    getindex(x::Nullable)
+
+Attempt to access the value of `x`. Throw a `NullException` if the value is not
+present. Usually, this is written as `x[]`.
+"""
+getindex(x::Nullable) = isnull(x) ? throw(NullException()) : x.value
+
+"""
+    get(x::Nullable)
+
+Alias for `getindex(x::Nullable)`.
```

Comment:
Personally, I found that docstring confusing. Perhaps we can wait for more feedback from others.